friends/src/App.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import { Route, Link } from "react-router-dom";
+import FriendsList from './components/FriendsList';
+import FriendForm from './components/FriendForm';
+
+class App extends Component {
+
+  constructor(){
+    super();
+    this.state ={
+      friends: [],
+      editFriend: {}
+    }
+  }
+
+
+componentDidMount(){
+      axios
+        .get('http://localhost:5000/friends')
+        .then(response => {
+          console.log(response);
+          this.setState({ friends: response.data });
+        })
+        .catch(err => {
+          console.log(err);
+        });
+    }
+
+    addToList = (obj) => {
```

Comment:
Maybe a more appropriate parameter name would be `friend`, or even `friendObj` if you want to know the data type for sure?